tests/fast-integration/eventing-test/eventing-test-prep.js

```diff
@@ -26,7 +26,7 @@ const {
   waitForEventingSinkFunction,
   deployV1Alpha1Subscriptions,
   deployV1Alpha2Subscriptions,
-  createK8sNamespace,
+  createK8sNamespace, isUpgradeJob,
```

Comment:
Teeny tiny nit: new line

Reply:
I was sooooo hoping you would not notice that .....